pset-functions.js

```diff
@@ -31,6 +31,22 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+function blackJack(playerCardScore, dealerCardScore) {
+   if (playerCardScore > dealerCardScore && playerCardScore <= 21) {
+      return playerCardScore
+   } else if (playerCardScore < dealerCardScore && dealerCardScore <= 21) {
+      return dealerCardScore
+   } else if (playerCardScore < dealerCardScore && playerCardScore <= 21) {
+      return playerCardScore
+   } else if (playerCardScore === 21 && dealerCardScore === 21) {
+      return 21
+   } else {
+      return 0
+   }
+}
+
+console.log(blackJack(19, 21))
+
 // Add your code above this line
```

Comment:
@cswebdesign Good Job her on this solution you have here. The function is good and you did a good job using the conditionals statements to get your outcome for the game.
Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L30)